
Header header

# Set either node_id or node_label
int32 node_id
string node_label

# optional: if not set, the base frame of the robot is used
string frame_id
